AzureDay.WebApp/AzureDay.WebApp.Notification/Email/Template/RestorePassword.tt

```diff
@@ -0,0 +1,17 @@
+﻿<#@ template language="C#" #>
```

Comment:
the whole file is redundant and should be removed, restore password will be handled by azure ad b2c